app/src/main/java/ca/klapstein/baudit/activities/MapAllProblemsActivity.java

```diff
@@ -87,12 +109,73 @@ public void onLowMemory() {
 
     @Override
     public void onMapReady(GoogleMap googleMap) {
-        LatLng ny = new LatLng(40.7143528, -74.0059731);
-        googleMap.moveCamera(CameraUpdateFactory.newLatLng(ny));
+        // LatLng ny = new LatLng(40.7143528, -74.0059731);
+        // googleMap.moveCamera(CameraUpdateFactory.newLatLng(ny));
+        // populateMap(presenter.getPatient(), googleMap);
+        testPopulateMap(googleMap);
     }
 
-    @Override
-    public void populateMap() {
-        // Do something
+    public void populateMap(Patient patient, GoogleMap googleMap) {
+        //https://developers.google.com/maps/documentation/android-sdk/map-with-marker
+        ProblemTreeSet problemTreeSet = patient.getProblemTreeSet();
+        for (Problem problem:problemTreeSet){
+            RecordTreeSet recordTreeSet = problem.getRecordTreeSet();
+            for(Record record: recordTreeSet){
+                LatLng marker = new LatLng(record.getGeoLocation().getLat(),record.getGeoLocation().getLon());
+                googleMap.addMarker(new MarkerOptions().position(marker).title(record.getTitle()));
+            }
+        }
+    }
+
+    public void testPopulateMap(GoogleMap googleMap){
```

Comment:
We should move tests into their own instrumented test files. If you want me to spearhead that let me know.